src/Server.zig

```diff
@@ -1596,6 +1596,21 @@ fn initializeHandler(server: *Server, writer: anytype, id: types.RequestId, req:
     log.info("zls initialized", .{});
     log.info("{}", .{server.client_capabilities});
     log.info("Using offset encoding: {s}", .{std.meta.tagName(server.offset_encoding)});
+
+    // TODO avoid having to call getZigEnv twice
+    // once in init and here
+    const env = Config.getZigEnv(server.allocator, server.config.zig_exe_path.?) orelse return;
+    defer std.json.parseFree(Config.Env, env, .{ .allocator = server.allocator });
+
+    const zig_build_exe_version = std.SemanticVersion.parse("0.10.0-dev.4177+6d7b0690a") catch return;
```

Comment:
Why is this value hardcoded? Is it meant to be changed upon a new website download release?

Reply:
This was my last minute testing code. Should have been more careful...